File: Java/src/Ejercicio8.java
import java.util.Scanner;

public class Ejercicio8{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = 0;
        int e = 0;
        int i =0;
        int o = 0;
        int u = 0;

        System.out.println("Introduce una palabra");
        String palabra = sc.nextLine();
        for (int j = 0; j < palabra.length(); j++){
            switch (palabra.charAt(j)){
                case 'a': a++; break;
                case 'e': e++; break;
                case 'i': i++; break;
                case 'o': o++; break;
                case 'u': u++; break;
                default: break;
            }
        }

        System.out.println("El numero de cada vocal de la palabra es: \n\ta: "+a+"\n\te: "+e+"\n\ti: "+i+"\n\to: "+o+"\n\tu: "+u);
        sc.close();
    }
}

File: Kotlin/src/Ejercicio8.kt
fun main() {
        var a = 0
        var e = 0
        var i = 0
        var o = 0
        var u = 0

        println("Introduce una palabra")
        val palabra = readln()
        for (j in 0 until palabra.length) {
            when (palabra[j]) {
                'a' -> a++
                'e' -> e++
                'i' -> i++
                'o' -> o++
                'u' -> u++
                else -> {}
            }
        }

        println("El numero de cada vocal de la palabra es: \n\ta: $a\n\te: $e\n\ti: $i\n\to: $o\n\tu: $u")
}